models/repository/repository.go

```diff
@@ -70,21 +70,21 @@ func (u *RepositoryStruct) Initialize() {
 	u.database = interfaceDB
 }
 
-func (u *RepositoryStruct) Save() bool {
+func (u *RepositoryStruct) Save() error {
```

Comment:
__[golint]__
* [golint] exported method RepositoryStruct.Save should have comment or be unexported